js/apps/admin-ui/src/user-federation/ldap/LdapSettingsGeneral.tsx

```diff
@@ -107,52 +107,38 @@ export const LdapSettingsGeneral = ({
         />
       )}
       <FormAccess role="manage-realm" isHorizontal>
-        <FormGroup
+        {/* These hidden fields are required so data object written back matches data retrieved */}
+        <KeycloakTextInput
+          hidden
+          id="kc-ui-provider-id"
```

Comment:
Lets get rid of the `id`.
```suggestion
          id="kc-ui-provider-id"
```